.azure-pipelines/templates/test-single-linux.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - template: './run-validations.yml'
 
   - ${{ if eq(parameters.test, 'true') }}:
-    - script: ddev test --cov ${{ parameters.check }}
+    - script: ddev test --cov --pytest-args --junitxml=junit/test-results.xml ${{ parameters.check }}
```

Comment:
Does that work ? Seems `--junitxml=junit/test-results.xml` will be considered as an argument of `ddev` command.

This worked for me: https://github.com/DataDog/integrations-core/pull/4591/files#diff-6ee732e856b9553ffa4de1c4242358d1R36

Reply:
It works locally at least.

Reply:
This works for me too but let's quote it please